lib/std/process.zig

```diff
@@ -2032,3 +2032,9 @@ pub fn createWindowsEnvBlock(allocator: mem.Allocator, env_map: *const EnvMap) !
     i += 1;
     return try allocator.realloc(result, i);
 }
+
+/// Logs an error and then terminates the process with exit code 1.
+pub fn fatal(comptime format: []const u8, format_arguments: anytype) noreturn {
```

Comment:
I don't understand this move. Wouldn't this be better on `std.log` itself? That is what Go does if that counts for anything https://pkg.go.dev/log#Fatal

Reply:
What another language does is entirely irrelevant, especially one as different as Go. Why would this live in `std.log`? Just because it happens to *use* that API doesn't mean it's *part* of it. `std.process` seems like an entirely reasonable place for this to live given its other task of exiting with code 1.

Reply:
Go and Zig both allow providing custom loggers. This is important because I would like to be able to call fatal on my custom logger without duplicating this function.

Reply:
if `std.log.fatal` was going to exist, i would still expect this one to exist and for `std.log.fatal` to call it

Reply:
I think it belongs in process since it exits the process.